static/js/settings_account.js

```diff
@@ -604,8 +602,8 @@ export function set_up() {
             html_submit_button: $t_html({defaultMessage: "Deactivate"}),
             on_click: handle_confirm,
             help_link: "/help/deactivate-your-account",
-            fade: true,
             loading_spinner: true,
+            disable_focus_yes_button: true,
```

Comment:
What's the story with removing the `fade` option?  

I think it's also worth extracting to a preparatory commit adding options like `disable_focus_yes_button` with an explanation for how you decided where it was needed; we ideally want the implementation change commit to not come with any changes to configuration.

Reply:
> What's the story with removing the fade option?

Consistency.
A user wouldn't differentiate between a "normal" modal and a "settings" modal. If one shows up instantly, one would expect all the others to do the same. The difference between Bootstrap `fade` and non-`fade` is pretty noticeable (300 ms for fading).
Besides, I am not aware of any app that does this.

In the case of Micromodal, I have set 120 ms for in and 75 ms for out, so it wouldn't feel slow.

> I think it's also worth extracting to a preparatory commit adding options like disable_focus_yes_button with an explanation for how you decided where it was needed; we ideally want the implementation change commit to not come with any changes to configuration.

That was an experiment. I forgot to remove it.

Reply:
Yeah, agreed.  The cleanest way to code that would be to just remove `fade` in a preparatory commit, so that the cutover commit doesn't have that complexity.